datanode/gateway/graphql/marshallers/ledgerentry_field.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2022 Gobalsky Labs Limited
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE.DATANODE file and at https://www.mariadb.com/bsl11.
+//
+// Change Date: 18 months from the later of the date of the first publicly
+// available Distribution of this version of the repository, and 25 June 2022.
+//
+// On the date above, in accordance with the Business Source License, use
+// of this software will be governed by version 3 or later of the GNU General
+// Public License.
+
+package marshallers
+
+import (
+	"fmt"
+
+	v2 "code.vegaprotocol.io/vega/protos/data-node/api/v2"
+	"github.com/99designs/gqlgen/graphql"
+)
+
+func MarshalLedgerEntryField(*v2.LedgerEntryField) graphql.Marshaler {
+	panic("not implemented")
+}
+
+func UnmarshalLedgerEntryField(i interface{}) (*v2.LedgerEntryField, error) {
+	v, ok := i.(string)
+	if !ok {
+		return nil, fmt.Errorf("expected string in ledger entry field")
```

Comment:
Can be a constant error.

Reply:
I am not sure where you mean to put this constant?

Reply:
Added a local var definition -> Addressed